packages/docs-utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui-internal/docs-utils",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
I guess this should be `1.0.0`?

Reply:
We are releasing this now from what I know, they are not private anymore. cc @michaldudak to confirm :)

Reply:
Yes, this needs to be bumped. The process isn't perfect yet, as this package has already been released outside of the normal Core releases, but we'll discuss this on the next code infra meeting